internal/core/server/workspace.go

```diff
@@ -704,8 +705,71 @@ func (ws *Workspace) DescribeProcesses(ctx context.Context, input *api.DescribeP
 				ID:       component.ID,
 				Name:     component.Name,
 				Provider: "docker",
-				Running:  state.Running,
 			}
+
+			containerInfo, err := ws.Docker.ContainerInspect(ctx, state.ContainerID)
+			if err == nil {
+				process.Running = containerInfo.State.Running
+				if process.Running {
+					statRequest, err := ws.Docker.ContainerStats(ctx, state.ContainerID, false)
+					if err != nil {
+						return nil, fmt.Errorf("could not get container stats: %w", err)
+					}
+					defer statRequest.Body.Close()
+
+					scanner := bufio.NewScanner(statRequest.Body)
```

Comment:
Why are you using a scanner here? It's a single JSON payload response, right? You can use `json.NewDecoder(statRequest.Body)`